apigw-websocket-api-lambda-authorizer/src/WebSocketAPI/Readme.md

```diff
@@ -0,0 +1,82 @@
+# WebSocket API Project
+
+This starter project consists of:
+* serverless.template - an AWS CloudFormation Serverless Application Model template file for declaring your Serverless functions and other AWS resources
```

Comment:
this line seems superfluous

Reply:
The project level README files are auto-generated when we create a project using Visual Studio. So, I have not touched it. Instead, I have created a main README file at root level.